scan/lib/scan/xcpretty_reporter_options_generator.rb

```diff
@@ -3,25 +3,29 @@
 module Scan
   class XCPrettyReporterOptionsGenerator
     SUPPORTED_REPORT_TYPES = %w(html junit json-compilation-database)
+    SUPPORTED_XCARGS_TYPES = %w(simple test tap knock)
```

Comment:
We allow other xcpretty formatters to be used with the `:format` option. Do other formatters allow other potential arg types? I don't want to limit this if other options are also usable 😊

Reply:
One more thing to add: for the format options, one of `simple`, `test`, `tap`, or `knock` can be used at a time, but these can be used together with `--report`, `--color`, etc. in the same execution.